iphone/cli/commands/_build.js

```diff
@@ -1265,7 +1269,13 @@ iOSBuilder.prototype.validate = function (logger, config, cli) {
 
 		// make sure the app doesn't have any blacklisted directories or files in the Resources directory and warn about graylisted names
 		var platformsRegExp = /^(android|ios|iphone|ipad|mobileweb|blackberry|windows|tizen)$/;
-		this.blacklistDirectories.push(this.tiapp.name);
+
+		if (this.blacklistDirectories.indexOf(this.tiapp.name.toLowerCase()) !== -1 || this.tiapp.name.toLowerCase() === 'frameworks') {
```

Comment:
I was unable to find evidence that `frameworks` was forbidden by Apple, so I graylist it and show a warning. We could blacklist it, but that would be a breaking change.